deploy/common/deploy.sh

```diff
@@ -78,23 +110,18 @@ function wait_for_central {
     local central_namespace=${2:-stackrox}
 
     echo -n "Waiting for Central in namespace ${central_namespace} to respond."
-    set +e
     local start_time
     start_time="$(date '+%s')"
     local deadline=$((start_time + 10*60))  # 10 minutes
-    until curl_central --output /dev/null --silent --fail "https://$LOCAL_API_ENDPOINT/v1/ping"; do
+    until curl_central_once --output /dev/null --fail "https://$LOCAL_API_ENDPOINT/v1/ping"; do
```

Comment:
Shouldn't this be with retries?

Reply:
I don't think it's necessary, this `until` loop serves as retrying. I think mixing a timeout-based and deadline-based stop conditions will cause more harm than good.